src/m365/commands/login.ts

```diff
@@ -107,13 +124,37 @@ class LoginCommand extends Command {
         case 'secret':
           auth.connection.authType = AuthType.Secret;
           auth.connection.secret = args.options.secret;
+          break;
+        case 'accessToken':
+          const accessTokens = typeof args.options.accessToken === "string" ? [args.options.accessToken] : args.options.accessToken as string[];
+          auth.connection.authType = AuthType.AccessToken;
+          auth.connection.appId = accessTokenUtil.accessToken.getTenantIdFromAccessToken(accessTokens[0]);
+          auth.connection.tenant = accessTokenUtil.accessToken.getAppIdFromAccessToken(accessTokens[0]);
+
+          for (const token of accessTokens) {
+            const resource = accessTokenUtil.accessToken.getAudienceFromAccessToken(token);
+            const expiresOn = accessTokenUtil.accessToken.getExpirationFromAccessToken(token);
+
+            auth.connection.accessTokens[resource] = {
+              expiresOn: expiresOn as Date || null,
```

Comment:
Should this be null or undefined?

Reply:
In the original typing of `AccessToken` it needs to be `string || Date || null`. That's the reason I'm using `|| null` here